openwisp_controller/notifications/apps.py

```diff
@@ -0,0 +1,59 @@
+from django.apps import AppConfig
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.translation import gettext_lazy as _
+from openwisp_notifications.types import (
+    register_notification_type,
+    unregister_notification_type,
+)
+
+
+class NotificationsConfig(AppConfig):
+    name = 'openwisp_controller.notifications'
+    label = 'controller_notifications'
+
+    def ready(self):
+        from . import handlers  # noqa
+
+        self.register_notification_types()
+
+    def register_notification_types(self):
+        register_notification_type(
+            'config_error',
+            {
+                'verbose_name': _('Configuration ERROR'),
+                'verb': _('encountered an error'),
+                'level': 'error',
+                'email_subject': _(
+                    '[{site.name}] ERROR: "{notification.target}" configuration'
+                    ' {notification.verb}'
+                ),
+                'message': _(
+                    'The configuration for [{notification.target}]'
+                    '({notification.target_link}) has {notification.verb}.'
+                    ' Last working configuration has been restored from the backup.'
```

Comment:
I'd move the space to the end of the previous line.

I'd use: "The last working configuration has been restored from a backup present on the filesystem of the device."